BUILD.bazel

```diff
@@ -11,6 +11,7 @@ exports_files([
 
 bzl_library(
     name = "bzl",
+    testonly = True,
```

Comment:
hmm, why?

Reply:
io_bazel_rules_webtesting has testonly on its bzl_library targets which feeds downstream to all rules using it; its not needed for all bzl_library targets in this repo but figured consistency is good

Reply:
nothing enforces that we do this (the rule that depends on these is not itself testonly), and these files are not used by tests. In theory a bzl_library could be used downstream by users who depend on our starlark API (not actually required to load a .bzl file today)
I don't see how we'll keep this consistent in the future, and I don't really care about being consistent with rules_webtesting if the sources aren't together.
I think I'd rather not have these

Reply:
There was an issue I ran into with not having them in a few places. I'll reproduce

Reply:
Here's the one under `/packages/karma/src/BUILD.bazel`:
```
bzl_library(
    name = "bzl",
    srcs = glob(["*.bzl"]),
    deps = [
        "@build_bazel_rules_nodejs//internal/common:bzl",
        "@build_bazel_rules_nodejs//internal/js_library:bzl",
        "@io_bazel_rules_webtesting//web",
    ],
)
```
depends on "@io_bazel_rules_webtesting//web" which has testonly = True so it needs it as well.

Reply:
And that feeds into the `stardoc` rule which depends on it:
```
stardoc(
    name = "docs",
    testonly = True,
    out = "index.md",
    input = "@npm_bazel_karma//:index.bzl",
    deps = [
        "@npm_bazel_karma//:bzl",
        # We need to restate local workspace dependencies here in `//foo:bzl`
        # format to work-around a bug in stardoc where .bzl files from
        # `@build_bazel_rules_nodejs//foo:bzl` style deps are not found
        # by the doc generator:
        # ```
        # Exception in thread "main" java.lang.IllegalStateException: File external/npm_bazel_karma/karma_node_test.bzl imported '@build_bazel_rules_nodejs//internal/common:devmode_js_sources.bzl', yet internal/common/devmode_js_sources.bzl was not found, even at roots [.].
        # ```
        "//internal/common:bzl",
        "//internal/js_library:bzl",
    ],
)
```

Reply:
The error for the above being:
```
ERROR: /private/var/tmp/_bazel_greg/35306ad70b8da8f7bc31590523de35c0/external/npm_bazel_karma/BUILD.bazel:54:1: in bzl_library rule @npm_bazel_karma//:bzl: non-test target '@npm_bazel_karma//:bzl' depends on testonly target '@io_bazel_rules_webtesting//web:web' and doesn't have testonly attribute set
ERROR: Analysis of target '//packages/karma:generate_README' failed; build aborted: Analysis of target '@npm_bazel_karma//:bzl' failed; build aborted
```

Reply:
Removed all the testonly's except for the two places they are necessary